README.md

```diff
@@ -45,10 +45,11 @@ Example of credentials file:
 Example execution:
 ```
 $ python lib/jira.py <tenant/managed> \
-  --url https://issues.redhat.com
-  --query 'project = KONFLUX AND status = "NEW" AND fixVersion = CY25Q1'
-  --credentials-file /path/to/credentials.json
-
+  --url https://issues.redhat.com \
+  --query 'project = KONFLUX AND status = "NEW" AND fixVersion = CY25Q1' \
+  --credentials-file /path/to/credentials.json \
+  --release /workspace/data/32a30aa8-203b-45b3-81cf-e3df7c0d7ed2/release.yaml \
+  --previousRelease /workspace/data/32a30aa8-203b-45b3-81cf-e3df7c0d7ed2/previous_release.yaml 
```

Comment:
Double check the implementation that Scott is doing, I think he passed us examples in YAML but they plan to send us the JSON instead.